pkgs/development/libraries/libvirt/default.nix

```diff
@@ -25,10 +25,10 @@ stdenv.mkDerivation rec {
     libxslt xhtml1 perlPackages.XMLXPath curl libpcap
   ] ++ stdenv.lib.optionals stdenv.isLinux [
```

Comment:
Instead of writing stdenv.lib throughout the nix file, add with stdenv.lib; at the top of the file and remove stdenv.lib throughout the file